lib/cc/config/validation/yaml.rb

```diff
@@ -29,6 +30,12 @@ def validate
           errors << "Unable to parse: #{ex.message}"
         end
 
+        def validate_version
+          if !data.key?("version") && (data.key?("plugins") || data.key?("exclude_paths"))
```

Comment:
Isn't `exclude_paths` a common key in version 1 configs?

Reply:
d'oh, I meant to type `exclude_patterns`. Fixing.